app/Handlers/Commands/Metric/AddMetricPointCommandHandler.php

```diff
@@ -36,7 +36,7 @@ public function handle(AddMetricPointCommand $command)
         ];
 
         if ($createdAt) {
-            $data['created_at'] = Carbon::createFromFormat('U', $createdAt)->format('Y-m-d H:i:s');
+            $data['created_at'] = Carbon::createFromFormat('U', $createdAt, config('cachet.timezone'))->format('Y-m-d H:i:s');
```

Comment:
i guess it would be "ok" to inject the config contract, but best would be to inject the actual config value, by registering a custom handler binding